plugins/woocommerce/templates/global/quantity-input.php

```diff
@@ -41,7 +41,7 @@
 		class="<?php echo esc_attr( join( ' ', (array) $classes ) ); ?>"
```

Comment:
Line 15 contains a `version` given we updated the template we should also update the template version number.
Could you change it to `@version 7.8.0` as this would get released in 7.8 when merged?

Reply:
@louwie17 I updated the version number in b1a87d0c162c6008b485928d6a89ae41adbe968c